sql/src/main/java/io/crate/protocols/postgres/SimplePortal.java

```diff
@@ -104,6 +105,9 @@ public Portal bind(String statementName,
                        List<Object> params,
                        @Nullable FormatCodes.FormatCode[] resultFormatCodes) {
 
+        if (locked()) {
+            throw new PortalAlreadyBoundException();
```

Comment:
there is no test for this case - I also think that throwing an error isn't the right thing to do.
The postgres protocol flow description states that unnamed portals are implicitly closed